Update Compaction topic doc

**BUG REPORT**

1. Please describe the issue you observed:

- What did you do (The steps to reproduce)?

```shell
$ bin/mqadmin updateTopic -w 8 -r 8 -a +delete.policy=COMPACTION -n localhost:9876 -t ctopic -c DefaultCluster
create topic to 127.0.0.1:10911 success.
TopicConfig [topicName=ctopic, readQueueNums=8, writeQueueNums=8, perm=RW-, topicFilterType=SINGLE_TAG, topicSysFlag=0, order=false, attributes={+delete.policy=COMPACTION}]
```

> 
      Caused by: org.apache.rocketmq.client.exception.MQClientException: CODE: 1  DESC: unsupported key: delete.policy
      For more information, please visit the url, https://rocketmq.apache.org/docs/bestPractice/06FAQ
	at org.apache.rocketmq.client.impl.MQClientAPIImpl.createTopic(MQClientAPIImpl.java:401)
	at 
      org.apache.rocketmq.tools.admin.DefaultMQAdminExtImpl.createAndUpdateTopicConfig(DefaultMQAdminExtImpl.java:270)
	at org.apache.rocketmq.tools.admin.DefaultMQAdminExt.createAndUpdateTopicConfig(DefaultMQAdminExt.java:189)
	at org.apache.rocketmq.tools.command.topic.UpdateTopicSubCommand.execute(UpdateTopicSubCommand.java:173)
	... 2 more

- What is expected to see?

The message property should be `cleanup.policy`

- What did you see instead?

2. Please tell us about your environment:

3. Other information (e.g. detailed explanation, logs, related issues, suggestions on how to fix, etc):

--- a/docs/cn/Example_Compaction_Topic_cn.md
+++ b/docs/cn/Example_Compaction_Topic_cn.md
@@ -13,7 +13,7 @@ $ bin/mqadmin updateNamesrvConfig -k orderMessageEnable -v true
 ```shell
 $ bin/mqadmin updateTopic -w 8 -r 8 -a +cleanup.policy=COMPACTION -n localhost:9876 -t ctopic -o true -c DefaultCluster
 create topic to 127.0.0.1:10911 success.
-TopicConfig [topicName=ctopic, readQueueNums=8, writeQueueNums=8, perm=RW-, topicFilterType=SINGLE_TAG, topicSysFlag=0, order=false, attributes={+delete.policy=COMPACTION}]
+TopicConfig [topicName=ctopic, readQueueNums=8, writeQueueNums=8, perm=RW-, topicFilterType=SINGLE_TAG, topicSysFlag=0, order=false, attributes={+cleanup.policy=COMPACTION}]
 ```
 
 ### 生产数据
@@ -28,7 +28,7 @@ producer.start();
 String topic = "ctopic";
 String tag = "tag1";
 String key = "key1";
-Message msg = new Message(topic, tag, key, "bodys"getBytes(StandardCharsets.UTF_8));
+Message msg = new Message(topic, tag, key, "bodys".getBytes(StandardCharsets.UTF_8));
 SendResult sendResult = producer.send(msg, (mqs, message, shardingKey) -> {
     int select = Math.abs(shardingKey.hashCode());
     if (select < 0) {
@@ -64,7 +64,7 @@ messageQueueList.forEach(mq -> {
 Map<String, byte[]> kvStore = Maps.newHashMap();
 while (true) {
     List<MessageExt> msgList = consumer.poll(1000);
-    if (msgList != null) {
+    if (CollectionUtils.isNotEmpty(msgList)) {
         msgList.forEach(msg -> kvStore.put(msg.getKeys(), msg.getBody()));
     }
 }

--- a/docs/en/Example_Compaction_Topic.md
+++ b/docs/en/Example_Compaction_Topic.md
@@ -12,7 +12,7 @@ $ bin/mqadmin updateNamesrvConfig -k orderMessageEnable -v true
 ```shell
 $ bin/mqadmin updateTopic -w 8 -r 8 -a +cleanup.policy=COMPACTION -n localhost:9876 -t ctopic -o true -c DefaultCluster
 create topic to 127.0.0.1:10911 success.
-TopicConfig [topicName=ctopic, readQueueNums=8, writeQueueNums=8, perm=RW-, topicFilterType=SINGLE_TAG, topicSysFlag=0, order=false, attributes={+delete.policy=COMPACTION}]
+TopicConfig [topicName=ctopic, readQueueNums=8, writeQueueNums=8, perm=RW-, topicFilterType=SINGLE_TAG, topicSysFlag=0, order=false, attributes={+cleanup.policy=COMPACTION}]
 ```
 
 ### produce message
@@ -25,7 +25,7 @@ producer.start();
 String topic = "ctopic";
 String tag = "tag1";
 String key = "key1";
-Message msg = new Message(topic, tag, key, "bodys"getBytes(StandardCharsets.UTF_8));
+Message msg = new Message(topic, tag, key, "bodys".getBytes(StandardCharsets.UTF_8));
 SendResult sendResult = producer.send(msg, (mqs, message, shardingKey) -> {
     int select = Math.abs(shardingKey.hashCode());
     if (select < 0) {
@@ -59,7 +59,7 @@ messageQueueList.forEach(mq -> {
 Map<String, byte[]> kvStore = Maps.newHashMap();
 while (true) {
     List<MessageExt> msgList = consumer.poll(1000);
-    if (msgList != null) {
+    if (CollectionUtils.isNotEmpty(msgList)) {
         msgList.forEach(msg -> kvStore.put(msg.getKeys(), msg.getBody()));
     }
 }

